dimagi/pages/templates/sections/pillar/mobile_data_collection/chapter4/part3.html

```diff
@@ -1,162 +1,46 @@
 {% load i18n %}
 {% load dimagidotcom %}
 
-<section class="epic-content pad-top-small pad-bottom-large"
-         id="how-program-environment-affect-tools">
+<section class="epic-content  pad-bottom-large"
+         id="chapter4-pilots">
   <div class="content content-width-medium">
 
     <h2>
       {% blocktrans %}
-        How Does Your Program and Its Environment
-        Affect Your Mobile Data Collection Tool?
+        Pilots
       {% endblocktrans %}
     </h2>
 
     <p>
       {% blocktrans %}
-        Mobile data collection tools range in price, some are harder
-        to maintain than others, and they all have different
-        strengths and weaknesses. Beyond the features that deal
-        with what data you collect, look at how the structure of
-        your program or the characteristics of the region you work
-        in might affect your platform needs. Here are some considerations:
+         Pilots are the time and place where we get our technology right and ensure we are developing something useful for the user. 
```

Comment:
one thing that I've seen in general for all of these PRs is that the text pasted in is in a really long line, and makes it difficult to read on an editor window that's half of your screen size. if this is possible, please adjust the copy so that it doesn't extend more than 80 characters in width. you can see in the github diff screen how it's forced to wrap nearly all of these lines of text, and this is not ideal.